### DISCUSSION TITLE
Site Localization for Non-Canonical Sites

### DISCUSSION THREAD

**Discussion by aad100** [vdemichev/DiaNN#1017] [Q&A]
Hello, I am trying to look for the possibility of non-canonical ubiquitination in DIA data.  Please can you let me know if it is possible to output site localisation confidence (PTM.Site.Confidence) for the non-canonical positions included in the search?  And if so how?  I have been using the following tags to try to achieve this:  --var-mod UniMod:121,114.042927,KTG  --strip-unknown-mods  --monitor-mod UniMod:21. Many thanks.

**Comment by vdemichev** [comment]
I've just realised I have this long unanswered question, sorry for the huge delay. Yes, it will work perfectly fine like this. Please try also DIA-NN 1.9, the localisation data will be in the main report in .parquet format.

Best,
Vadim
